src/components/app-header.ts

```diff
@@ -1,7 +1,8 @@
 export enum EPanelId {
   controlPanel = "control-panels",
   dashboard = "dashboard",
-  workspace = "workspace"
+  workspace = "workspace",
+  stats = "stats"
```

Comment:
In the absence of any other sensible order I'd keep these in alphabetical order.

Reply:
Fixed